src/scenario/scenario_events_xml.c

```diff
@@ -0,0 +1,530 @@
+#include "scenario_events_xml.h"
+
+#include "assets/assets.h"
+#include "core/array.h"
+#include "core/buffer.h"
+#include "core/calc.h"
+#include "core/file.h"
+#include "core/image_group.h"
+#include "core/io.h"
+#include "core/lang.h"
+#include "core/log.h"
+#include "core/string.h"
+#include "core/xml_parser.h"
+#include "core/zlib_helper.h"
+#include "empire/city.h"
+#include "empire/type.h"
+#include "scenario/scenario_event.h"
+#include "scenario/scenario_event_data.h"
+#include "scenario/scenario_events_controller.h"
+#include "scenario/scenario_events_parameter_data.h"
+#include "window/plain_message_dialog.h"
+#include <math.h>
```

Comment:
Space between internal and external defines, please.

Reply:
Done.